Hub/Dockerfile.txt

```diff
@@ -1,3 +1,5 @@
+USER seluser
+
```

Comment:
Running all commands as `seluser`, so the ENV vars a present for this user always.